Formula/swi-prolog.rb

```diff
@@ -41,13 +45,11 @@ def install
       end
     end
 
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args,
-                      "-DSWIPL_PACKAGES_JAVA=OFF",
-                      "-DSWIPL_PACKAGES_X=OFF",
-                      "-DCMAKE_INSTALL_PREFIX=#{libexec}"
-      system "make", "install"
-    end
+    args = ["-DSWIPL_PACKAGES_JAVA=OFF", "-DSWIPL_PACKAGES_X=OFF"]
+    args << "-DCMAKE_INSTALL_RPATH=@loader_path" if OS.mac?
```

Comment:
Not sure of best way to handle this. Error was:
```
Variable-referenced libraries:
  @rpath/libswipl.8.dylib
Files with missing rpath:
  /usr/local/Cellar/swi-prolog/8.4.2_1/libexec/lib/swipl/lib/x86_64-darwin/archive4pl.so
```

The dylib is located in same directory `/usr/local/Cellar/swi-prolog/8.4.2_1/libexec/lib/swipl/lib/x86_64-darwin/libswipl.8.dylib`.

Just added `@loader_path` to RPATH for now.